### DISCUSSION TITLE
Latest Snap Release (2.7.1) on Stable Branch rev1552 vs. 1563

### DISCUSSION THREAD

**Discussion by newrath** [keepassxreboot/keepassxc#7963] [Q&A] [answered]
Hi all, 
I saw on snap that on stable branch a new revision (2.7.1) has been published although the version number remains the same. In this package the manifest and snapcraft yaml files are missing.
The revision is same as on edge branch - is this intended or has it been pushed wrongly on stable?

**Comment by droidmonkey** [comment, marked as answer]
This is intentional. It is to correct issues with the file browser on some distros.
